manifests/base/redis/redis-secrets.env

```
@@ -0,0 +1,2 @@
+# change the password
+redis-password='redis_password'
```

Env variable are always in upper case and with `_` feels weird to see them with lower case and `-`
```suggestion
REDIS_PASSWORD='redis_password'
```